modules/common/users/accounts.nix

```diff
@@ -34,7 +34,7 @@ in
 
   config = mkIf cfg.enable {
     users = {
-      mutableUsers = true;
+      mutableUsers = false;
```

Comment:
Sounds like a good default anyway. I am also evaluating userborn as an alternative to the perl script just now.